### PR TITLE
Add difference geometry operation

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

File: doc/developer/webservices.rst [hunk 1]
```diff
@@ -240,3 +289,26 @@ Result
 
 The 'Content-Type' header is 'text/csv',
 and the data contains the given 'csv' data.
+
+Geometry operations
+===================
+
+This service gives geometry operations (actually only one)
```
Comment (asaunier) [inline doc/developer/webservices.rst]: actually => currently

**Comment by asaunier** [comment]
Looks good

File: c2cgeoportal/tests/functional/test_geometry_operation.py [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    tear_down_common as tearDownModule,
+    set_up_common as setUpModule,
+    create_dummy_request
+)
+
+
+@attr(functional=True)
+class TestGeometryOperation(TestCase):
+
+    def test_defference(self):
```
Comment (pgiraud) [inline c2cgeoportal/tests/functional/test_geometry_operation.py]: typo e -> i

File: doc/developer/webservices.rst (hunk 1 above)
Comment (pgiraud) [inline doc/developer/webservices.rst]: gives => provides

File: doc/developer/webservices.rst [hunk 3]
```diff
@@ -240,3 +289,26 @@ Result
...
+
+Method: ``POST``
+
+Data:
+
+.. code:: json
+
+   {
+       "geom": <The geom to extrude>
+       "diff": <The geom to extrude with>
```
Comment (pgiraud) [inline doc/developer/webservices.rst]: What about geomA & geomB?

**Comment by pgiraud** [comment]
According to postGIS doc, ST_Difference is a processing, not an operation.

File: doc/developer/webservices.rst (hunk 3 above)
Comment (sbrunner) [inline doc/developer/webservices.rst]: For me geomA & geomB mean nothing, then I try to use better names :-)

**Comment by sbrunner** [comment]
Review applyed :-)

File: c2cgeoportal/tests/functional/test_geometry_processing.py [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+
+from c2cgeoportal.tests.functional import (  # noqa
+    tear_down_common as tearDownModule,
+    set_up_common as setUpModule,
+    create_dummy_request
+)
+
+
+@attr(functional=True)
+class TestGeometryOperation(TestCase):
```
Comment (pgiraud) [inline c2cgeoportal/tests/functional/test_geometry_processing.py]: Please change Operation to Processing.

File: doc/developer/webservices.rst (hunk 3 above)
Comment (pgiraud) [inline doc/developer/webservices.rst]: But it's compliant with the PostGIS doc. http://postgis.net/docs/ST_Difference.html
Also, we often see schemas where A & B names are used.
text4247

Even if not perfect, I find geomA and geomB easier to understand. I would be interested to know what others think.

File: c2cgeoportal/tests/functional/test_geometry_processing.py (hunk 4 above)
Comment (sbrunner) [inline c2cgeoportal/tests/functional/test_geometry_processing.py]: Oups :-)

File: doc/developer/webservices.rst (hunk 3 above)
Comment (sbrunner) [inline doc/developer/webservices.rst]: on Sqlite it's geom1, geom2
on argis, geometry1, geometry2
on OGC: g1, g2 (http://portal.opengeospatial.org/files/?artifact_id=25354)

then I can put g1, g2 event if I think that's worth ...

Comment (sbrunner) [inline doc/developer/webservices.rst]: Finally I think that the different names are so bad that I can have something like this:
``` json
{
    arguments: [<geomA>, <geomB>]
}
```
What do you think ?

Comment (pgiraud) [inline doc/developer/webservices.rst]: What about "geometries" or "geoms" instead of "arguments".

Comment (sbrunner) [inline doc/developer/webservices.rst]: OK pour `"geometries"` :-)

**Comment by sbrunner** [comment]
Updated :-)

**Pull request merged**